src/Middleware/integrations/OrderCloud.Integrations.Alerts/OrderCloud.Integrations.Alerts.csproj

```diff
@@ -11,12 +11,6 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.16.0" />
-    <PackageReference Include="ordercloud-dotnet-catalyst" Version="2.0.1" />
-    <PackageReference Include="OrderCloud.SDK" Version="0.11.0" />
-    <PackageReference Include="StyleCop.Analyzers" Version="1.1.118">
-      <PrivateAssets>all</PrivateAssets>
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-    </PackageReference>
```

Comment:
Why did you remove StyleCop?

Reply:
Same reason as the other packages. It was added to a bunch of projects instead of inherited from Common.

Reply:
Its not an important change, I kind of got swept up in a "cleaning" mood

Reply:
Ah, I see now the other changes. 

That is actually by design, see[ this architecture discussion](https://github.com/ordercloud-api/headstart/issues/414). Basically though, we don't want those lower level foundational packages to rely on the higher level Headstart.Common.

Reply:
Ah, my bad. So I should revert all the deleted packages? And update the versions everwhere.

Reply:
Yes please